internal/cmd/profile/usage_view_csv_command.go

```diff
@@ -0,0 +1,108 @@
+package profile
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/spacelift-io/spacectl/internal/cmd"
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
+)
+
+type queryArgs struct {
+	since   string
+	until   string
+	aspect  string
+	groupBy string
+}
+
+func usageViewCSVCommand() *cli.Command {
+	return &cli.Command{
+		Name:      "usage-view-csv",
+		Usage:     "Prints CSV with usage data for the current account",
+		ArgsUsage: cmd.EmptyArgsUsage,
+		Flags: []cli.Flag{
+			flagUsageViewCSVSince,
+			flagUsageViewCSVUntil,
+			flagUsageViewCSVAspect,
+			flagUsageViewCSVGroupBy,
+			flagUsageViewCSVFile,
+		},
+		Before: authenticated.Ensure,
+		Action: func(ctx *cli.Context) error {
+			// prep http query
+			args := &queryArgs{
+				since:   ctx.String(flagUsageViewCSVSince.Name),
+				until:   ctx.String(flagUsageViewCSVUntil.Name),
+				aspect:  ctx.String(flagUsageViewCSVAspect.Name),
+				groupBy: ctx.String(flagUsageViewCSVGroupBy.Name),
+			}
+			params := buildQueryParams(args)
```

Comment:
Is `queryArgs` struct really needed? You could just pass the `ctx *cli.Context` and parse it there